viewer-config-persistence/src/main/resources/scripts/hqsqldb-add_can_create_new_relation.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE feature_type_relation
+    ADD COLUMN can_create_new_relation BOOLEAN DEFAULT FALSE NOT NULL
```

Comment:
dit levert ander gedrag op dan in de vorige versie na een upgrade, niet? ik zou denken dat de default `true` moet zijn